Sources/AblyAssetTrackingInternal/WorkerQueue/Workers/LegacyWorker.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+/// A worker that runs tasks that the Publisher and Subscriber used to run asynchronously on their
+/// own dispatch queues. This worker allows us to tie up legacy work and new asynchronous
+/// work, for simplicity and consistency during the transition.
+open class LegacyWorker<PropertiesType, WorkerSpecificationType> : Worker
```

Comment:
Does this class need to be generic? It doesn't make use of properties or specifications. I think we can remove these generic parameters and change the usage of `PropertiesType` and `WorkerSpecificationType` inside this class to `Void`.

Reply:
Ah – sorry, I think I am wrong on this one, let me take another look.

Reply:
Yes, ignore me, I'm talking nonsense on this one, needs to be generic.

Reply:
👍🏻